Gemfile

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
+ruby IO.read(File.expand_path('.ruby-version', __dir__)).strip
```

Comment:
This will prohibit deployment until the installed ruby on the server matches (because Capistrano won't be able to bundle).